WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Warning/JetpackRestoreWarningViewController.swift

```diff
@@ -79,7 +79,7 @@ class JetpackRestoreWarningViewController: UIViewController {
             self?.dismiss(animated: true)
         }
 
-        warningView.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .systemBackground
```

Comment:
I made a mistake in the previous PR using `safeArea` to pin the content view but it resulted in a clear navigation bar – this fixes it.